entrypoint.sh

```diff
@@ -171,28 +162,28 @@ if ! test -d /etc/sv/dnsmasq ; then
     bogus-priv
 DNSMASQ
 
-    mkdir /etc/sv/dnsmasq
-    cat <<RUNIT >/etc/sv/dnsmasq/run
+  mkdir -p /etc/service/dnsmasq
+  cat <<RUNIT >/etc/service/dnsmasq/run
 #!/bin/sh
 exec /usr/sbin/dnsmasq --no-daemon
 RUNIT
-    chmod +x /etc/sv/dnsmasq/run
+  chmod +x /etc/service/dnsmasq/run
 
-# dnsmasq service log
-    mkdir /etc/sv/dnsmasq/log
-    mkdir /etc/sv/dnsmasq/log/main
-    cat <<RUNIT >/etc/sv/dnsmasq/log/run
+  # dnsmasq service log
+  mkdir -p /etc/service/dnsmasq/log
+  mkdir -p /etc/service/dnsmasq/log/main
+  cat <<RUNIT >/etc/service/dnsmasq/log/run
 #!/bin/sh
 exec svlogd -tt ./main
 RUNIT
-    chmod +x /etc/sv/dnsmasq/log/run
-    ln -s /etc/sv/dnsmasq /etc/service/dnsmasq
+  chmod +x /etc/service/dnsmasq/log/run
+  ln -s /etc/service/dnsmasq /etc/service/dnsmasq
```

Comment:
This line is not required, as `/etc/service/dnsmasq` is the actual directory now